tools/targets/__init__.py

```diff
@@ -379,6 +379,21 @@ def labels(self):
         labels = (names + CORE_LABELS[self.core] + self.extra_labels)
         return labels
 
+    @property
+    def core_without_NS(self):
+        if self.core.endswith('-NS'):
+            return self.core[:-3]
+        else:
+            return self.core
+
+    @property
+    def is_TrustZone_secure_target(self):
+        return 'TFM' in self.labels and not self.core.endswith('-NS')
+
+    @property
+    def is_TrustZone_non_secure_target(self):
+        return self.core.endswith('-NS')
```

Comment:
```suggestion
        return not self.is_TrustZone_secure_target
```

Reply:
No, because it's not the opposite of secure - there are 3 cases:

* TrustZone secure target
* TrustZone non-secure target
* non-TrustZone target (neither of the above)